src/auxiliary/precompile.jl

```diff
@@ -382,10 +382,10 @@ function _precompile_manual_()
     # end
     # end
     @assert Base.precompile(Tuple{typeof(SummaryCallback)})
-    @assert Base.precompile(Tuple{DiscreteCallback{typeof(Trixi.summary_callback),
-                                                   typeof(Trixi.summary_callback),
-                                                   typeof(Trixi.initialize_summary_callback),
-                                                   typeof(SciMLBase.FINALIZE_DEFAULT)}})
+    # @assert Base.precompile(Tuple{DiscreteCallback{typeof(Trixi.summary_callback),
+    #                                                typeof(Trixi.summary_callback),
+    #                                                typeof(Trixi.initialize_summary_callback),
+    #                                                typeof(SciMLBase.FINALIZE_DEFAULT)}})
```

Comment:
Will the CI fail if you uncomment these lines?

Reply:
Tried in https://github.com/trixi-framework/Trixi.jl/pull/2150/commits/fdf1ff566847d87751caf7920d850b9833db85d0; lets see

Reply:
Thanks